src/components/ChooseCheckGroup.tsx

```diff
@@ -40,6 +41,10 @@ export const ChooseCheckGroup = () => {
 const CheckGroupCard = ({ group }: { group: CheckTypeGroupOption }) => {
   const styles = useStyles2(getStyles);
   const { isOverCheckLimit, isOverScriptedLimit } = useLimits();
+  const checkOptions = useCheckTypeOptions().filter((option) => option.group === group.value);
+  const checksWithStatus = checkOptions.filter((option) => option.status);
+  const shouldShowStatus = checksWithStatus.length === checkOptions.length;
```

Comment:
This actually resulted in the Public Preview badge appearing for Scripted checks which I thought was a good thing!

![A screenshot of the Scripted and Browser check cards next to each other. The scripted card has a green Public preview badge beneath it. The Browser card has a purple Private preview badge beneath it.](https://github.com/user-attachments/assets/d3653a67-eeec-436f-8165-042c4ff91966)